skills/bear-from-alps/SKILL.md

```
@@ -0,0 +1,1158 @@
+
+1. Ask for the ALPS profile path
+2. Read the file and verify it is valid ALPS JSON
+3. Validate with `asd --validate {profile_path}`
+
+```bash
+# Validation command
+asd --validate docs/alps.json
+```
+
+**On validation failure:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)